File: python/1097_sequencia_ij_3.py
I = 1
J = 7
contador = 0

while I <= 9:
    print(f'I={I} J={J}')
    J -= 1
    contador += 1

    if contador == 3:
        I += 2
        J = 7
        contador = 0

    if I > 9:
        break

File: c/c99/1 - Iniciante/1097_sequencia_ij_3.c
#include <stdio.h>

int main() {
    int I = 1, J = 7, contador = 0;

    while (I <= 9) {
        printf("I=%d J=%d\n", I, J);
        J -= 1;
        contador += 1;

        if (contador == 3) {
            I += 2;
            J += 5;
            contador = 0;
        }
    }
    return 0;
}
